pkg/proxy/ipvs/proxier.go

```diff
@@ -556,7 +560,7 @@ func cleanupIptablesLeftovers(ipt utiliptables.Interface) (encounteredError bool
 	for _, tc := range tableChainsWithJumpService {
 		if err := ipt.DeleteRule(tc.table, tc.chain, args...); err != nil {
 			if !utiliptables.IsNotFoundError(err) {
-				glog.Errorf("Error removing pure-iptables proxy rule: %v", err)
+				glog.Errorf("Error removing ipvs Proxier iptables rule: %v", err)
```

Comment:
Nit: "Error removing iptables rule in ipvs proxier" sounds a little better. You can make this change in multiple places.

Reply:
Sounds fair, it's fixed now.